core/src/main/scala/cats/Parallel.scala

```diff
@@ -0,0 +1,139 @@
+package cats
+
+import cats.arrow.FunctionK
+
+/**
+  * Some types that form a Monad, are also capable of forming an Applicative that supports parallel composition.
+  * The Parallel type class allows us to represent this relationship.
+  */
+trait Parallel[M[_], F[_]] extends Serializable {
+  /**
+    * The applicative instance for F[_]
+    */
+  def applicative: Applicative[F]
+
+  /**
+    * Natural Transformation from the parallel Applicative F[_] to the sequential Monad M[_].
+    */
+  def sequential(implicit M: Monad[M]): F ~> M
+
+  /**
+    * Natural Transformation from the sequential Monad M[_] to the parallel Applicative F[_].
+    */
+  def parallel(implicit M: Monad[M]): M ~> F
+}
+
+object Parallel extends ParallelArityFunctions {
+
+  def apply[M[_], F[_]](implicit P: Parallel[M, F]): Parallel[M, F] = P
+
+  /**
+    * Like `Traverse[A].sequence`, but uses the applicative instance
+    * corresponding to the Parallel instance instead.
+    */
+  def parSequence[T[_]: Traverse, M[_]: Monad, F[_], A]
+  (tma: T[M[A]])(implicit P: Parallel[M, F]): M[T[A]] = {
+    implicit val F = P.applicative
+    val fta: F[T[A]] = Traverse[T].traverse(tma)(P.parallel.apply)
+    P.sequential.apply(fta)
```

Comment:
if we remove the `Monad` implicit arg (and make it a implicit def in the typeclass), we can do `P.sequential(fta)` here right?  That would be a better API.